src/components/common/ViewerSidebar.jsx

```diff
@@ -673,19 +682,45 @@ class ViewerSidebar extends Component {
                                 </div>,
                                 'annotation',
                                 'annotation',
-                                newCellTypeAnnoTableOboCell
+                                newCellTypeAnnoTableOboCell,
+                                (a, b) => {
+                                    if (
+                                        a.annotation_label ===
+                                        b.annotation_label
+                                    ) {
+                                        return 0;
+                                    }
+                                    return a.annotation_label >
+                                        b.annotation_label
+                                        ? 1
+                                        : -1;
+                                }
```

Comment:
Add `import * as R from 'rambda'`;
```suggestion
                                R.comparator((a, b) => a.annotation_label < b.annotation_label)
```